resources/sdk/clisdkclient/extensions/experimental_features/experimental_features.go

```diff
@@ -20,6 +21,9 @@ func IsValidCommand(command string) bool {
 	if command == "dummy_command" {
 		return true
 	}
+	if command == "alternative_formats" {
```

Comment:
You can remove this bit. The check above is only necessary for the dummy_command